### DISCUSSION TITLE
Inlined/Top label for Dropdown (notch?)

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [primefaces/community#1825] [PrimeReact] [answered]
![Untitled](https://github.com/primefaces/community/assets/10191085/a90ff992-ab01-4421-b332-7551e435be13)
![Untitled](https://github.com/primefaces/community/assets/10191085/baafb968-2bd2-4dfe-aaf5-fc769711418f)

I'm helping someone with React, but I don't work with React, I work with Vue.
Anyway, I'm wondering if the Dropdown component or any other component for selecting options, can achieve the inlined/top label (notch?).

Thanks.

**Comment by melloware** [comment, marked as answer]
I am not sure if any of the PrimeReact themes support that "notched" label style on FloatLabel: https://primereact.org/floatlabel/

**Reply by angelhdzmultimedia** [reply]
Thanks. MUI has it and the person I'm helping was already using MUI.

I'm gonna start learning PrimeVue soon. 🔥
